data/data/manifests/bootkube/openshift-kubevirt-infra-namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Remove this, it moved to MCO https://github.com/openshift/machine-config-operator/blob/master/install/0000_80_machine-config-operator_00_namespace.yaml

Reply:
@ravidbro ?

Reply:
@rgolangh that is new and unlike the other platforms.
We were asked not to add a new namespace to the MCO repo, so we added it here.

Reply:
https://github.com/openshift/machine-config-operator/pull/2098#discussion_r519750323